src/app.c

```diff
@@ -213,18 +218,28 @@ bool MTY_WindowSetUITexture(MTY_App *app, MTY_Window window, uint32_t id, const
 		return false;
 
 	MTY_Device *device = gfx_ctx_get_device(cmn);
+	if (!device)
+		return false;
```

Comment:
Avoid a bunch of error scenarios by not allowing the function to proceed if the graphics context doesn't have a good device handle. This scenario occurs when the GPU resets.